trunner/runners/IMXRT117xRunner.py

```diff
@@ -0,0 +1,45 @@
+#
+# Phoenix-RTOS test runner
+#
+# IMXRT117x runner
+#
+# Copyright 2021 Phoenix Systems
+# Authors: Damian Loewnau, Jakub Sarzyński
+#
+
+import time
+
+from .ARMV7M7Runner import ARMV7M7Runner
+
+
+class IMXRT117xRunner(ARMV7M7Runner):
+    """This class provides interface to run test case on IMXRT117x using RaspberryPi.
+
+    Default configuration for running on Rpi:
+
+    GPIO 17 must be connected to the JTAG_nSRST (j21-15) (using an additional resistor 1,5k).
+
```

Comment:
I think we don't need double-newline with each new sentence.

Reply:
I thought it looks better in a preview, like here. But I think without it it looks better in code, changed